docker-compose.yml

```diff
@@ -6,6 +6,8 @@ services:
         restart: always
         depends_on:
             - db
+        volumes:
```

Comment:
To jest po to, aby widzieć logi na serwerze testowym. Nie da się podglądać logów na dockerze, bo nie ma zainstalowanego edytora tekstowego.